build.cake

```diff
@@ -159,8 +160,14 @@ Task("PackBinaries")
 {
     var actions = new List<Action>();
 
-    actions.Add(() => DoPackage("Calamari", "net40", nugetVersion));
+    actions.Add(() => DoPackage("Calamari", "net40", nugetVersion));    
     actions.Add(() => DoPackage("Calamari", "net452", nugetVersion, "Cloud"));
+    
+    // make nuget packages out of Common and CloudAccounts so that they can be used by Sashimi packages
+    actions.Add(() => DoPackage("Calamari.Common", "net40", nugetVersion));
+    actions.Add(() => DoPackage("Calamari.Common", "net452", nugetVersion));
+    actions.Add(() => DoPackage("Calamari.Common", "netstandard2.1", nugetVersion));
+    actions.Add(() => DoPackage("Calamari.CloudAccounts", "net452", nugetVersion));
```

Comment:
Doesn't `Calamari.CloudAccounts` also need `netstandard2.1`?

Reply:
Pretty sure this packages is referenced in Sashimi

Reply:
You're right @johnsimons. Sorry, I specifically went and checked this too, but it was last thing on a Friday afternoon so I must have overlooked something.

Reply:
Good catch - yes it would.